backend/OpenAPI.yaml

```diff
@@ -57,24 +56,117 @@ paths:
             application/json:
               schema:
                 type: object
-                required:
-                  - error
-                properties:
-                  error:
-                    type: string
-                    example: "Invalid email format. Please use a UofT email."
+              properties:
+                error_code:
+                  type: string
+                  example: "INVALID_INPUT"
+                message:
+                  type: string
+                  example: "Invalid email or password"
+                details:
+                  type: string
+                  example: "Email must be a valid University of Toronto email"
+
         "409":
           description: Conflict - Email already in use
+          content:
+            application/json:
+            schema:
+              type: object
+            properties:
+              error_code:
+                type: string
+                example: "EMAIL_ALREADY_IN_USE"
+              message:
+                type: string
+                example: "The email address is already registered."
+              details:
+                type: string
+                example: "Please use a different email address of log in."
+        "500":
```

Comment:
The error schemas should probably be aligned together with those from Qais' PR so we can discuss this tonight.